octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/util/icon/EntityIconFactory.java

```diff
@@ -85,26 +88,66 @@ private void init() {
 
     private void loadImageData(Entity entity) {
         IconDetail iconDetail = iconDetailMap.containsKey(entity) ? iconDetailMap.get(entity) : undefinedIconDetail;
-
+           
         Display display = Display.getDefault();
+        Color white = display.getSystemColor(SWT.COLOR_WHITE);
         Image img = new Image(display, iconWidth, iconHeight);
         GC gc = new GC(img);
-
+ 
+        gc.setBackground(white);
```

Comment:
is setting the background white because we don't support other themes in eclipse?

Reply:
The logic behind this is to create an image with a separate color as background which is then assigned to be transparent, it is needed to be made white in order to work in the editor part logo but even with this the icons work on multiple themes